src/Services/FHIR/FhirValueSetService.php

```diff
@@ -116,30 +114,35 @@ public function getAll($fhirSearchParameters, $puuidBind = null): ProcessingResu
     {
         $fhirSearchResult = new ProcessingResult();
         try {
-              // Start with postcalendar_categories as appointment-type
-              $calendarCategories = $this->appointmentService->getCalendarCategories($fhirSearchParameters);
-              $valueSet = new FHIRValueSet();
-              $valueSet->setId('appointment-type');
-              $compose = new FHIRValueSetCompose();
-              $include = new FHIRValueSetInclude();
-            foreach ($calendarCategories as $category) {
-                if ($category["pc_cattype"] != 0) {
-                    continue; // only cat_type==0
+            if (!isset($fhirSearchParameters[ '_id' ]) || $fhirSearchParameters[ '_id' ] == 'appointment-type') {
```

Comment:
While there maybe put a comment here for us so others know.

Reply:
Actually, array is AND and that will never work for multiple values, but the spec also includes comma delimited OR so that's now implemented.  I wasn't grokking the method from the trait so I coded this explicitly with comments.